src/plugins/analysis/users_and_passwords/code/password_file_analyzer.py

```diff
@@ -21,7 +21,7 @@
 USER_NAME_REGEX = br'[a-zA-Z][a-zA-Z0-9_-]{2,15}'
 UNIX_REGEXES = [
     USER_NAME_REGEX + br':[^:]?:\d+:\d*:[^:]*:[^:]*:[^\n ]*',
-    USER_NAME_REGEX + br':\$[1256][ay]?\$[a-zA-Z0-9\./+]+\$[a-zA-Z0-9\./+]{16,128}={0,2}',  # MD5 / Blowfish / SHA
+    USER_NAME_REGEX + br':\$[1256][ay]?\$[a-zA-Z0-9\./+]*\$[a-zA-Z0-9\./+]{16,128}={0,2}',  # MD5 / Blowfish / SHA
```

Comment:
These regexes deserve some documentation. Despite the commit message it is not clear to me why this change makes it possible to leave out the salt.
Maybe we should at least document that they are used to parse entrys to `/etc/passwd`.

Reply:
> Maybe we should at least document that they are used to parse entrys to /etc/passwd.

that is what `UNIX_REGEXES` should imply but we could also rename it to `UNIX_PASSWD_REGEXES` to make it more clear

Reply:
>  rename it to UNIX_PASSWD_REGEXES to make it more clear

What about `ETC_PASSWD_REGEXES`?
But I dont have a strong opinion on that. Go ahead and merge if you think this is good to go.